pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -662,13 +678,16 @@ EOF
 
 echo "enabling aro services"
 for service in aro-dbtoken aro-monitor aro-portal aro-rp auoms azsecd azsecmond mdsd mdm chronyd fluentbit; do
-  systemctl enable $service.service
+	systemctl enable $service.service
 done
 
 for scan in baseline clamav software; do
-  /usr/local/bin/azsecd config -s $scan -d P1D
+	/usr/local/bin/azsecd config -s $scan -d P1D
 done
 
 echo "rebooting"
 restorecon -RF /var/log/*
-(sleep 30; reboot) &
+(
```

Comment:
This is one example of a style change that I think we can leave out of the rpVMSS.sh script unless it's needed. We're creating differences between the gateway and RP scripts by only editing one, and not sure it's required for your changes.

Reply:
This was unintentionally done by one of my linters and I have removed them.